sdk/core/core-http/src/httpHeaders.ts

```diff
@@ -82,6 +82,7 @@ export interface HttpHeadersLike {
   toJson(): RawHttpHeaders;
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
definitely a place to use `unknown` I think

Reply:
Done.